pom.xml

```diff
@@ -7,7 +7,7 @@
 
     <groupId>net.juniper.netconf</groupId>
     <artifactId>netconf-java</artifactId>
-    <version>2.1.1.6-SNAPSHOT</version>
+    <version>2.1.1.6.2-eks</version>
```

Comment:
Is this final code? or still in the draft? what is 'eks'?

Reply:
I added the `-eks` so that I could reference this code in my own project without intersecting with your published artifacts (at least until you merge in my changes). Yes this is in a final state